docs/en/stack/ml/df-analytics/ml-dfa-outlier-detection.asciidoc

```diff
@@ -101,16 +99,15 @@ altered data.
 [[dfa-feature-influence]]
 === Feature influence
 
-Besides the {olscore}, another value is calculated during {oldetection}: 
-the feature influence score. As we mentioned, there are multiple features of a 
-data point that are analyzed during {oldetection}. An influential feature is a 
-feature of a data point that is responsible for the point being an outlier. The 
-value of feature influence provides a relative ranking of features by their 
-contribution to a point being an outlier. Therefore, while {olscore} tells us 
-whether a data point is an outlier, feature influence shows which features make 
-the point an outlier. By doing this, this value provides context to help 
-understand more about the reasons for the data point being unusual and can drive 
-visualizations.
+Besides the {olscore}, another value is calculated during {oldetection}: the 
```

Comment:
paragraph can be reworded as "Feature influence, another score calculated while detecting outliers, provides a relative ranking of the different features and their contribution towards a point being an outlier. This score allows an end user to understand the context or reasoning on why a certain data point is an outlier"